src/actions.js

```diff
@@ -75,6 +75,7 @@ export default {
         ),
       );
   },
+  // @@@ should this really be something within scaife-widgets?
   [FETCH_LIBRARY]: ({ commit }) => {
```

Comment:
@paltman I'm thinking that rather than having the `FETCH_LIBRARY` action implemented at the site level, it makes more sense for the GraphQL query to live in scaife-widgets:

[LibraryWidget](https://github.com/scaife-viewer/scaife-widgets/blob/65562539ea12a1902225d6d4089b78bcbcb633d3/src/widgets/LibraryWidget.vue#L21)

(similar to [PassageAncestorsWidget](https://github.com/scaife-viewer/scaife-widgets/blob/65562539ea12a1902225d6d4089b78bcbcb633d3/src/widgets/PassageAncestorsWidget.vue#L25)

What do you think?

Reply:
@jacobwegner I completely agree.

Reply:
Great–I'll work on factoring it out